arch/arm/src/tlsr82/Make.defs

```diff
@@ -0,0 +1,92 @@
+############################################################################
+# arch/tc32/src/tlsr82/Make.defs
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+############################################################################
+
+include common/Make.defs
+
+# Specify our HEAD assembly file.  This will be linked as
+# the first object file, so it will appear at address 0
+HEAD_ASRC  = cstartup_flash.S
+
+# Arch Flags
+AFLAGS += -DMCU_STARTUP_FLASH
+
+# Filter-out unnecessary .S files
+CMN_ASRCS :=
+
+# Filter-out unnecessary .c files
+TLSR82_CSRCS_FILTER := arm_backtrace_fp.c arm_backtrace_thumb.c arm_hostfs.c
+TLSR82_CSRCS_FILTER += arm_etherstub.c arm_udelay.c arm_semi_syslog.c
+TLSR82_CSRCS_FILTER += arm_fullcontextrestore.c arm_saveusercontext.c
+TLSR82_CSRCS_FILTER += arm_switchcontext.c arm_vfork.c
```

Comment:
don't need filter out arm_etherstub.c and arm_udelay.c since PR https://github.com/apache/incubator-nuttx/pull/6208.
and arm_hostfs.c, arm_semi_syslog.c and  arm_vfork.c.

Reply:
Done